ember-intl/addon/helpers/format-date.ts

```diff
@@ -12,7 +12,7 @@ type Options = Params[1];
 interface FormatDateSignature {
   Args: {
     Named?: Options & { allowEmpty?: boolean };
-    Positional: [Value?, Options?];
+    Positional: [Value?] | [Value | null | undefined, Options];
```

Comment:
This fixes a type error that had slipped through `v6.0.0-beta.3`, when we started to allow passing a POJO as the 2nd argument.

https://ember-intl.github.io/ember-intl/versions/v6.3.0/docs/guide/migration-5-0-to-6-1#dynamically-set-named-arguments

Reply:
**TECHNICAL RISK**

To my understanding,`Options` is a combination of an options hash supported by `@formatjs/intl` and a user-defined hash that provides data.

So, we may be unintentionally allowing end-developers to provide `@formatjs/intl` as a part of their user-defined hash.